server/src/internalClusterTest/java/org/elasticsearch/cluster/routing/allocation/decider/WriteLoadConstraintDeciderIT.java

```diff
@@ -451,10 +810,12 @@ private Settings enabledWriteLoadDeciderSettings(int utilizationThresholdPercent
                 WriteLoadConstraintSettings.WRITE_LOAD_DECIDER_HIGH_UTILIZATION_THRESHOLD_SETTING.getKey(),
                 utilizationThresholdPercent + "%"
             )
-            // TODO (ES-12862): remove these overrides when throttling is turned off for simulations.
-            .put(CLUSTER_ROUTING_ALLOCATION_NODE_CONCURRENT_INCOMING_RECOVERIES_SETTING.getKey(), 100)
-            .put(CLUSTER_ROUTING_ALLOCATION_NODE_CONCURRENT_OUTGOING_RECOVERIES_SETTING.getKey(), 100)
-            .put(CLUSTER_ROUTING_ALLOCATION_NODE_CONCURRENT_RECOVERIES_SETTING.getKey(), 100)
+            .put(
+                WriteLoadConstraintSettings.WRITE_LOAD_DECIDER_QUEUE_LATENCY_THRESHOLD_SETTING.getKey(),
+                TimeValue.timeValueMillis(queueLatencyThresholdMillis)
+            )
+            // Essentially disable rebalancing so that testing can see Decider change outcomes.
+            .put(BalancedShardsAllocator.THRESHOLD_SETTING.getKey(), 1000f)
```

Comment:
Nit: Could you use `EnableAllocationDecider#CLUSTER_ROUTING_REBALANCE_ENABLE_SETTING` to be more explicit?